### PR TITLE
SCRUM-62

### PR DESCRIPTION
**Pull request by Remma27**
https://ever-utn.atlassian.net/browse/SCRUM-62?atlOrigin=eyJpIjoiMjYzYjljODMyZDFhNGYyYTgwMTUwYjAyNjVkOTMzMjAiLCJwIjoiaiJ9

### CODE REVIEW

File: swagger.yml
Comment (ebarahonautn) [inline swagger.yml:1]: @Remma27 
Revisar, arreglar y asegurar que el swagger.yml este actualizado con respecto a los endpoints.

Screenshot 2024-06-24 at 9 52 01 AM

**Review by ebarahonautn — CHANGES_REQUESTED** [review]

File: swagger.yml
Comment (ebarahonautn) [inline swagger.yml:1]: @Remma27 
POST endpoint replace _description_ by **location**

Screenshot 2024-06-24 at 8 37 33 PM

**Review by ebarahonautn — CHANGES_REQUESTED** [review]

File: swagger.yml
Comment (ebarahonautn) [reply swagger.yml:1]: PR tiene conflictos

Comment (ebarahonautn) [inline swagger.yml:1]: @Remma27 

Please change the current response for GetAll and GetbyId with the correct attributes and check the conflict in the PR.

{
      "_id": "0183IEHU#&#J",
      "name": "Laboratories",
      "location": "Laboratories Zone"
    }

Screenshot 2024-06-25 at 5 02 02 PM
Screenshot 2024-06-25 at 5 02 18 PM

**Review by ebarahonautn — APPROVED** [review]

**Review by XxKeyDarkxX — APPROVED** [review]

**Pull request merged**